src/tesseract.pybind.h

```diff
@@ -70,15 +70,35 @@ void add_tesseract_module(py::module &root) {
   py::class_<TesseractDecoder>(m, "TesseractDecoder")
       .def(py::init<TesseractConfig>(), py::arg("config"))
       .def("decode_to_errors",
-           py::overload_cast<const std::vector<uint64_t> &>(&TesseractDecoder::decode_to_errors),
+           py::overload_cast<const std::vector<uint64_t>&>(&TesseractDecoder::decode_to_errors),
            py::arg("detections"))
       .def("decode_to_errors",
-           py::overload_cast<const std::vector<uint64_t> &, size_t, size_t>(
+           py::overload_cast<const std::vector<uint64_t>&, size_t, size_t>(
                &TesseractDecoder::decode_to_errors),
            py::arg("detections"), py::arg("det_order"), py::arg("det_beam"))
-      .def("mask_from_errors", &TesseractDecoder::mask_from_errors, py::arg("predicted_errors"))
+      .def(
+          "mask_from_errors",
+          [](TesseractDecoder& self, const std::vector<size_t>& predicted_errors) {
+            std::vector<bool> result(self.num_observables, false);
+            const auto& indices = self.mask_from_errors(predicted_errors);
+            for (int index : indices) {
+              result[index] = true;
+            }
```

Comment:
Rather than getting the indices first (which requires set operations) it might be more efficient to iterate over the errors directly in this routine (and likewise for the analogous simplex method):
```
  for (size_t ei : predicted_errors) {
    for (int obs_index : self.errors[ei].symptom.observables) {
        result[obs_index] ^= true;
    }
  }
```